ROSNET/DataModel/Message.cs

```diff
@@ -1,20 +1,44 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Linq;
+using ROSNET.Field;
 
 namespace ROSNET.DataModel
 {
+    /// <summary>
+    /// Represents a ROSbag message
+    /// </summary>
     public class Message
     {
-        public int ConnID { get; }
-        public long Time { get;  }
-        private byte[] Data { get; }
+        //Header fields of message record:
+        public int Conn { get; }
+        public (uint, uint) Time { get;  }
```

Comment:
Jeg ville laget en egen klasse for Time, ettersom det er litt mer logikk som må på plass for å få noe nyttig ut av de to intsene. Så lag en klasse Time, som har de to intsene som felt, og lag en methode i klassen for å få ut tiden i form av en DateTime/DateTimeOffset

Reply:
Ville faktisk brukt en `struct` gjerne til og med `readonly record struct` om dere går til C# 10 i samme slengen (som dere 100% burde gjøre), siden det er en spesifikk verdi